test/clientPolicies.spec.ts

```diff
@@ -1,21 +1,43 @@
 // tslint:disable:no-unused-expression
 import * as chai from 'chai';
 import {KeycloakAdminClient} from '../src/client';
+import ClientProfilesRepresentation from '../src/defs/clientProfilesRepresentation';
 import {credentials} from './constants';
 
 const expect = chai.expect;
 
 describe('Client Policies', () => {
   let kcAdminClient: KeycloakAdminClient;
+  let currentClientProfiles: ClientProfilesRepresentation;
 
   before(async () => {
     kcAdminClient = new KeycloakAdminClient();
     await kcAdminClient.auth(credentials);
   });
 
   it('lists client policy profiles', async () => {
-    const profiles = await kcAdminClient.clientPolicies.listProfiles();
+    const profiles = await kcAdminClient.clientPolicies.listProfiles(
+      {includeGlobalProfiles: true},
+    );
     expect(profiles).to.be.ok;
+    currentClientProfiles = profiles;
+  });
+
+  it('create client policy profiles', async () => {
+    const globalProfiles = currentClientProfiles.globalProfiles;
+    const newClientProfiles = {
+      "profiles": [
+        {
+          "name": "test",
+          "executors": []
+        }
+    ], globalProfiles }
+
+   const createdClientProfile = await kcAdminClient.clientPolicies.createProfiles(
+     newClientProfiles,
+   );
+   
+   expect(createdClientProfile).to.be.deep.eq('');
```

Comment:
This returns an empty string? Strange.

Reply:
yeah, this is strange, I was expecting it to be an empty [], not ''.

Reply:
I guess that is just how the API works :man_shrugging: